Common/src/interface_interpolation/CSlidingMesh.cpp

```diff
@@ -245,10 +245,10 @@ void CSlidingMesh::SetTransferCoeff(const CConfig* const* config) {
 
           dTMP = 0;
           for(iDim = 0; iDim < nDim; iDim++){
-            target_iMidEdge_point[iDim] = ( TargetPoint_Coord[ target_segment[0] ][ iDim ] +
-                                            target_geometry->nodes->GetCoord( target_iPoint , iDim) ) / 2;
-            target_jMidEdge_point[iDim] = ( TargetPoint_Coord[ target_segment[1] ][ iDim ] +
-                                            target_geometry->nodes->GetCoord( target_iPoint , iDim) ) / 2;
+            target_iMidEdge_point[iDim] = ( TargetPoint_Coord(target_segment[0], iDim ) +
+                                            target_geometry->nodes->GetCoord( target_iPoint , iDim) ) / 2.;
```

Comment:
I am curious: Did you get a compiler notification or so on this `2.` or was it just personal preference?

Reply:
Personal preference. Once I spent a lot of time finding out that 1/2==0.